bin/publish

```diff
@@ -27,4 +28,19 @@ if [ "$GIT_DESCRIPTION" = "v${VERSION}" ]; then
     docker tag "$IMAGE_NAME:$FULL_VERSION_TAG" "$REGISTRY/$IMAGE_NAME:$tag"
     docker push "$REGISTRY/$IMAGE_NAME:$tag"
   done
+
+  # Publish only latest to Redhat Registries
+  echo "Tagging and pushing ${REDHAT_IMAGE}"
+  docker tag "secrets-provider-for-k8s-redhat:${FULL_VERSION_TAG}" "${REDHAT_IMAGE}:${VERSION}"
+
+  if docker login scan.connect.redhat.com -u unused -p "${REDHAT_API_KEY}"; then
+    # you can't push the same tag twice to redhat registry, so ignore errors
+    if ! docker push "${REDHAT_IMAGE}:${VERSION}"; then
+      echo 'RedHat push FAILED! (maybe the image was pushed already?)'
+      exit 0
+    fi
+  else
+    echo 'Failed to log in to scan.connect.redhat.com'
```

Comment:
@sigalsax will this fail the build? how would we know that the published failed?

Reply:
correct. I forgot the `exit 1`